.github/workflows/publish-next.yml

```diff
@@ -156,8 +156,18 @@ jobs:
     secrets:
       NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-  build-test:
+  deploy-sample-app:
+    runs-on: ubuntu-latest
+    environment: deployment
     needs: publish
+    steps:
+      - name: Trigger build for beta liveness sample app pointing at next tag
+        run: curl -X POST -d {} $ENDPOINT -H "Content-Type:application/json"
+        env:
+          ENDPOINT: ${{ secrets.LIVENESS_BETA_SAMPLE_APP_BUILD_TRIGGER }}
```

Comment:
This env var needs to be added to deployment environment, @wlee221 does this seem like a sensible change?